_upgrade-to/upgrade-to.md

```diff
@@ -169,7 +169,7 @@ If you are migrating an Open Distro for Elasticsearch cluster, we recommend firs
    # Security plugin disabled
    curl -XGET 'localhost:9200/_nodes/_all?pretty=true'
    # Security plugin enabled
-   curl -XGET -k -u 'admin:admin' 'https://localhost:9200/_nodes/_all?pretty=true'
+   curl -XGET -k -u 'admin:< Admin password >' 'https://localhost:9200/_nodes/_all?pretty=true'
```

Comment:
```suggestion
   curl -XGET -k -u 'admin:<custom-admin-password>' 'https://localhost:9200/_nodes/_all?pretty=true'
```

Reply:
Applied suggestion in: [cdeb90f](https://github.com/opensearch-project/documentation-website/pull/5887/commits/cdeb90ffae8be7a3ae2f9de267abc08879dd44cd)